src/System.Management.Automation/engine/CommandBase.cs

```diff
@@ -286,8 +286,6 @@ public enum ActionPreference
         Inquire,
         /// <summary>Ignore the event completely (not even logging it to the target stream)</summary>
         Ignore,
-        /// <summary>Suspend the command for further diagnosis. Supported only for workflows.</summary>
-        Suspend,
```

Comment:
I am not sure - that will break compiling with SDK.

Reply:
I removed it based on [this comment](https://github.com/PowerShell/PowerShell/issues/9570#issuecomment-492842811). If necessary I can put it back, but that also would mean adding back logic to raise runtime errors if it is used. Better to remove it IMHO.

Also correct me if I'm wrong, but wouldn't breaking compiling with the SDK be a good thing in this case since workflow is out and the enumeration is no longer used, so code that uses it would need to be updated?

Reply:
> but that also would mean adding back logic to raise runtime errors if it is used.

No need to raise runtime errors. 
My thoughts is that the "Suspend" value is in public type and removing it will be a breaking change for scripts and codes. If we remove this we lost _binary_ backward compatibility too. 
There is 3 option: (1) Remove the value, (2) add Obsolete attribute, (3) add only comment that the value is "Obsolete".
Last option is my preference _today_ to avoid a breaking change with the public type. Similar used in CoreFX for some types so that we can continue to compile PowerShell without errors - it is not critical, related code is silently ignored. Also the CoreFX practice allows us to use new runtimes with previously compiled code.

Reply:
I was thinking earlier this evening about the obsolete attribute. Seems with that I can decide if it generates compiler errors or warnings, right? If we generate warnings, that would preserve backwards compatibility while giving more visibility to the change, no?

Reply:
> If we generate warnings ...

It is not always so. If the user uses "warnings as errors" compiler option (we use it in the repo), then this does not work - the user will not be able to compile their project. That's why there is no use of this attribute in CoreFX.

Reply:
The committee feedback was to keep the enum, so this has been addressed.